src/rust/ide/json-rpc/src/error.rs

```diff
@@ -28,6 +27,11 @@ pub enum RpcError {
     /// Failed to deserialize message from server.
     #[fail(display = "Failed to deserialize a message: {}.", _0)]
     DeserializationFailed(serde_json::Error),
+
+    /// Response timeout.
+    #[allow(missing_docs)]
+    #[fail(display = "Response timeout after {} ms.", millis)]
+    TimeoutError{millis:u128},
```

Comment:
Why we use u128 here? Do we expect timeout exceeding 18446744073709551615 (c.a. 500 millions years)?

Reply:
It's the type that is returned by Duration.